brands/shop/clothes.json

```diff
@@ -182,11 +182,25 @@
     }
   },
   "shop/clothes|Bonobo": {
+    "countryCodes": ["fr"],
+    "nomatch": ["shop/clothes|Bonobos"],
     "tags": {
       "brand": "Bonobo",
+      "brand:wikidata": "Q63682093",
+      "clothes": "women",
```

Comment:
https://www.bonoboplanet.com/ seems to carry also clothes for men ("homme" in French translates to "man").